x-pack/plugins/cases/public/components/connectors/resilient/case_fields.test.tsx

```diff
@@ -100,13 +102,13 @@ describe('ResilientParamsFields renders', () => {
       </MockFormWrapperComponent>
     );
 
-    const checkbox = within(screen.getByTestId('incidentTypeComboBox')).getByTestId(
+    const checkbox = within(await screen.findByTestId('incidentTypeComboBox')).getByTestId(
```

Comment:
should within(await screen.findByTestId('incidentTypeComboBox')).**getByTestId**( be findBy?